web/source/keymanweb.js

```diff
@@ -897,7 +897,10 @@
           return null;
         }
 
-        keymanweb.inputList.push(Pelem['kmw_ip']); 
+        keymanweb.inputList.push(Pelem['kmw_ip']);
+        
+        console.log("Unexpected state - this element's simulated input DIV should have been removed from the page!");
```

Comment:
We shouldn't ever get here in the present form of the code.  This _is_ a very recent change, however, as old versions of my solution used to leave the simulated input element intact with the DOM document in case the element were ever re-added to the page.  The idea _had_ been to reuse the element and save ourselves some trouble, but that turned out to have numerous issues.

The if-block in which this code lies is fully intended to help in handling cases where a previously-removed element is re-added to the page, for extreme dynamic cases.